### PR TITLE
Audio Analysis Endpoint

### PR DESCRIPTION
**Pull request by Grabajuice**

### CODE REVIEW

File: lib/src/models/audio_analysis.dart [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+part of spotify.models;
+
+@JsonSerializable(createToJson: false)
+class AudioAnalysis extends Object {
+  AudioAnalysis();
+
+  factory AudioAnalysis.fromJson(Map<String, dynamic> json) =>
+      _$AudioAnalysisFromJson(json);
+
+List<TimeInterval> bars;
```
Comment (rinukkusu) [inline lib/src/models/audio_analysis.dart:13]: Please some more whitespace in this file ;)

File: lib/src/models/section.dart [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  /// The confidence, from 0.0 to 1.0, of the reliability of the tempo. Some tracks contain tempo changes or sounds which don’t contain tempo (like pure speech) which would correspond to a low value in this field.
+  double tempo_confidence;
+
+  ///The estimated overall key of the section. The values in this field ranging from 0 to 11 mapping to pitches using standard Pitch Class notation (E.g. 0 = C, 1 = C♯/D♭, 2 = D, and so on). If no key was detected, the value is -1.
+  int key;
+
+  /// The confidence, from 0.0 to 1.0, of the reliability of the key. Songs with many key changes may correspond to low values in this field.
+  double key_confidence;
+
+  ///	Indicates the modality (major or minor) of a track, the type of scale from which its melodic content is derived. This field will contain a 0 for “minor”, a 1 for “major”, or a -1 for no result. Note that the major key (e.g. C major) could more likely be confused with the minor key at 3 semitones lower (e.g. A minor) as both keys carry the same pitches.
```
Comment (rinukkusu) [inline lib/src/models/section.dart:37]: Again with the long comments - you know the drill :)

**Review by rinukkusu — CHANGES_REQUESTED** [review]
Again, looks great over all, and again please remove the merge commit from the change list. Wondering how that happened 🤔 

Thank you!

**Comment by hayribakici** [comment]
@rinukkusu I tried to fix the merge conflicts. I didn't know that it would inflict 79 files. Sorry about this mess. Any ideas on how to fix this?

**Comment by rinukkusu** [comment]
@hayribakici I think there is nothing we can do really, with @Grabajuice not stepping into action. Probably easier to take the diff from this commit https://github.com/rinukkusu/spotify-dart/pull/93/commits/15adcf70e9666a04f42187b0d87f33a4d2a8e524 and reintegrate the code in a new branch from master.

**Pull request closed without merging**